rules/linux/lnx_system_network_discovery.yml

```diff
@@ -0,0 +1,28 @@
+title: System Network Discovery - Linux
+id: e7bd1cfa-b446-4c88-8afb-403bcd79e3fa
+status: experimental
+description: Detects enumeration of local network configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    category: process_creation
+    product: unix
+detection:
+    selection:
+        ProcessName:
+            - '/usr/bin/firewall-cmd'
+            - '/usr/sbin/ufw'
+            - '/usr/sbin/iptables'
+            - '/usr/bin/netstat'
+            - '/usr/bin/ss'
+            - '/usr/sbin/ip'
+            - '/usr/sbin/ifconfig'
```

Comment:
sorry for getting to it after the approval. 
I just recalled that paths to the executables could vary from one distribution to another.
I've just checked path to the `ss` on an Ubuntu machine, and it was `/bin/ss`.

So it makes sense to apply the following changes:

```
        ProcessName|endswith:
            - '/firewall-cmd'
            - '/ufw'
            - '/iptables'
            - '/netstat'
            - '/ss'
            - '/ip'
            - '/ifconfig'
```

Could you also please adjust the rest of your Linux rules following this scenario?
Sorry for the inconvenience.

Reply:
No worries, thanks for taking time to review this work. Please see the updated commit, it should include your recommendations.

Reply:
Thank you very much!